addons/hr_recruitment/wizard/applicant_refuse_reason.py

```diff
@@ -16,6 +16,7 @@ class ApplicantGetRefuseReason(models.TransientModel):
         domain="[('model', '=', 'hr.applicant')]")
     applicant_without_email = fields.Text(compute='_compute_applicant_without_email',
         string='Applicant(s) not having email')
+    applicant_email = fields.Text(compute='_compute_applicant_email')
```

Comment:
Maybe `applicants_emails`? because we are talking about more that one applicant and more than one email